src/main/java/io/vertx/proton/impl/ProtonConnectionImpl.java

```diff
@@ -23,6 +23,7 @@
 import java.util.Map;
 import java.util.UUID;
 
+import io.vertx.proton.*;
```

Comment:
Using IntelliJ IDEA as IDE, it adds import and when they overcome a specific number from same package (I don't know what this number is), it uses *.